src/Components/ActivationKey/AdditionalRepositoriesCard.js

```diff
@@ -49,20 +50,21 @@ const AdditionalRepositoriesCard = (props) => {
             BaseOS and AppStream, are always enabled and do not need to be
             explicitly added to the activation key.
           </Text>
-          <AddAdditionalRepositoriesButton
-            onClick={handleEditAdditionalRepositoriesToggle}
-          />
-          <AddAdditionalRepositoriesModal
-            isOpen={isEditAdditionalRepositoriesModalOpen}
-            handleModalToggle={handleEditAdditionalRepositoriesToggle}
-            keyName={activationKey.name}
-            repositories={availableRepositories}
-            isLoading={isLoading}
-            error={error}
-          />
         </TextContent>
+        <AddAdditionalRepositoriesButton
+          onClick={handleEditAdditionalRepositoriesToggle}
+        />
+        <AddAdditionalRepositoriesModal
+          isOpen={isEditAdditionalRepositoriesModalOpen}
+          handleModalToggle={handleEditAdditionalRepositoriesToggle}
+          keyName={activationKey.name}
+          repositories={availableRepositories}
+          isLoading={isLoading}
+          error={error}
+        />
```

Comment:
Is there a reason this was moved outside of the `textContent`? That changes the spacing a little